src/internalBinding/timers.cc

```diff
@@ -36,21 +36,14 @@ static bool enqueueWithDelay(JSContext *cx, unsigned argc, JS::Value *vp) {
   return true;
```

Comment:
I think we should return an Int32. This would also be closer to the specs: https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#return_value

and consistent with your comment in test_event_loop.py
setTimeout should return a positive integer value

Reply:
@philippedistributive I believe I put `double` instead of `int32` here for a reason. I couldn't remember why.

Reply:
changed to
```c
args.rval().setNumber(PyEventLoop::AsyncHandle::getUniqueId(std::move(handle)));
```
to match https://github.com/Distributive-Network/PythonMonkey/blob/e4c40acb6f161e7161aba929e02c99a55a1dcd38/src/internalBinding/timers.cc#L42

Reply:
[`.setNumber()`](https://github.com/mozilla/gecko-dev/blob/151afcc/js/public/Value.h#L673-L719)